src/extension.ts

```diff
@@ -488,8 +488,10 @@ export class Extension implements RunHooks {
           return;
 
         const trace = result.attachments.find(a => a.name === 'trace')?.path || '';
+        const prevTrace = (testItem as any)[traceUrlSymbol];
```

Comment:
Relying on the previous file is a bit fragile, but given that there is no clear way to get currently selected tree item, I'm ok with it. Let's add a comment explaining what is going on.